droid-hal-device.inc

```diff
@@ -299,7 +290,6 @@ The recovery.img for device
 %if 0%{?bootloader_binary:1}
 ################
 %package aboot
-Group:  System
 Summary: Bootloader for %{device} hw
 %if 0%{?enable_bootloader_update:1}
 # The device flashing info and scripts are in droid-configs
```

Comment:
Below there is a newline missing before `%description` in straggler-files section, for 100% tidyness :)
https://github.com/mer-hybris/droid-hal-device/blob/c4325dceb1e435cf8eeae7d5dea1a6ab43dd38c1/droid-hal-device.inc#L309